src/mumble/Log.cpp

```diff
@@ -705,17 +707,29 @@ void Log::log(MsgType mt, const QString &console, const QString &terse, bool own
 	if ((flags & Settings::LogConsole)) {
 		QTextCursor tc = Global::get().mw->qteLog->textCursor();
 
+		tc.movePosition(QTextCursor::End);
+
+		// A newline is inserted after each frame, but this spaces out the
+		// log entries too much, so the font size is set to near-zero in order
+		// to reduce the space between log entries. This font size is set only
+		// for the blank lines between entries, not for entries themselves.
+		static QTextCharFormat cf = tc.blockCharFormat();
```

Comment:
Why are you using a static variable for this?

Reply:
QTextCursor's charFormat and blockCharFormat methods return the char format for the current cursor position. Since the cursor can get moved (i.e. when the user selects text in the chat window) I thought it might help to make sure we get the char format for the empty log widget and not from some message. However, since this assignment happens immediately after moving the cursor to the bottom of the log, it's probably unnecessary.

Reply:
yeah, I think it should be fine without `static` :thinking: